packages/integrations/image/src/build/ssr.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs/promises';
+import path from 'path';
+import glob from 'tiny-glob';
+import { fileURLToPath } from 'url';
+import { ensureDir } from '../utils/paths.js';
+
+async function globImages(dir: URL) {
+	const srcPath = fileURLToPath(dir);
+	return await glob(
+		`${srcPath}/**/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}`,
+		{ absolute: true }
+	);
+}
+
+export interface SSRBuildParams {
+	srcDir: URL;
+	outDir: URL;
+}
+
+export async function ssrBuild({ srcDir, outDir }: SSRBuildParams) {
+	const images = await globImages(srcDir);
```

Comment:
SSR build step moved out of the main integration

This was actually a bug uncovered while fixing the undefined error, original images weren't being copied to dist for SSR